YAML/sublime-syntax.sublime-syntax

```diff
@@ -0,0 +1,532 @@
+%YAML 1.2
+---
+# http://www.sublimetext.com/docs/3/syntax.html
+name: YAML sublime-syntax
+file_extensions:
+  - sublime-syntax
+first_line_match: ^%YAML( 1.\d)?$
+scope: source.yaml
+
+variables:
+  variable: '(\{\{)([a-zA-Z_0-9]*)(\}\})'
+  escaped_char: \\(?>[0atnvfre "/\\N_Lp]|x[0-9a-fA-F]{1,2}|x\{[0-9a-fA-F]{1,8}\}|u\d{4}|U\d{8}|c\d+|C-\d+|M-\d+|M-\\C-\d+)
+
+contexts:
+
```

Comment:
one more suggestion :) to correctly scope the yaml directive (the very first line of the file) using `- include: scope:source.yaml#directive`